### ISSUE TITLE
dropping partial support for htmlwidget

### ISSUE THREAD

**Issue by kulbhushanchand** [kulbhushanchand/kulbhushanchand.github.io-v3#2]
The theme's original layout does not support `dt` and `plotly` html widgets. There was the conflict of loading jquery before `dt` and `plotly` htmlwidget scripts, which breaks their functionality.

For this issue, the main.js was made to load after the footer (actually after loading the htmlwidgets script files) and the conflicting htmlwidgets works fine.

However there is a new issue of https://github.com/kulbhushanchand/kulbhushanchand.github.io/issues/1 and also I don't plan to use the `dt` and `plotly` in the near future, which makes me rethink to revert the theme's original layout.

My original intention for using htmlwidgets is primarily to use the `leaflet` package, which fortunately works fine with my [changes to the knitr function](https://github.com/kulbhushanchand/kulbhushanchand.github.io/commit/f5c870be4d0fcf3604588b132e4fc3adbd13ff5a).

Tl:dr - I don't find `dt` and `plotly` useful enough to warrant changes in the theme original files.
